CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+# 3.1.0
+> Nov 15, 2017
+* :nut_and_bolt: **New** Add prettier-eslint 💅
+* :nut_and_bolt: **New** Support react 15.x || 16.x 🕺🏼
```

Comment:
Please link this point to the registered issue (https://github.com/SparebankenVest/react-css-collapse/issues/14)